bin/resin-preload

```diff
@@ -132,6 +131,12 @@ buildOutputStream.pipe(process.stdout)
 
 streamToPromise(buildOutputStream)
 .then(() => {
+  if (options.apiToken) {
+    return resin.token.set(options.apiToken)
+  }
+  return Promise.resolve()
```

Comment:
You don't need this line - doing nothing (implicitly `return undefined`) should work just as well.